ThunderCloud/ContentController.swift

```diff
@@ -1272,9 +1284,10 @@ public extension ContentController {
         if var imageSize = lastUnderScoreComponent {
             
             // Replace these for a later check
-            imageSize = imageSize.replacingOccurrences(of: ".jpg", with: "")
-            imageSize = imageSize.replacingOccurrences(of: ".png", with: "")
-            
+            extensions.forEach { (fileExtension) in
+                imageSize = imageSize.lowercased().replacingOccurrences(of: fileExtension, with: "")
```

Comment:
Extremely unlikely, but would break images with `.png` in their name. Could maybe ignore this one

Reply:
It will I think... but I think we're safe to ignore for now!